api/TransitionEvent.json

```
@@ -161,10 +182,10 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": null
+              "version_added": true
             },
             "firefox_android": {
-              "version_added": null
+               "version_added": true
```

I've been having this problem with GitHub introducing spurious white space changes into suggestions sometimes. Sorry about that

```suggestion
              "version_added": true
```